languages/go/exercises/concept/comments/.docs/hints.md

```diff
@@ -0,0 +1,22 @@
+## General
+
+- [Documentation comments][comment] should be written directly before the thing that they are describing, start with the name of what they are describing, take the form of a sentence, and end with a period.
```

Comment:
I'm not a native english speaker but maybe `entity` fits?

```suggestion
- [Documentation comments][comment] should be written directly before the entity that they are describing, start with the name of what they are describing, take the form of a sentence, and end with a period.
```

Reply:
`entity` sounds good! :)